automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -245,7 +245,7 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
 
     "should install JupyterLab" taggedAs Tags.SmokeTest in {
       withProject { project => implicit token =>
-        withNewCluster(project, request = ClusterRequest(userJupyterExtensionConfig = Some(jupyterLabExtensionClusterRequest))) { cluster =>
+        withNewCluster(project, request = ClusterRequest()) { cluster =>
```

Comment:
This test is interesting because we now install lab by default. Maybe we should reduce the test so that it just checks that the /lab URL is accessible, but doesn't try to loc/deloc (since that is covered by other tests).